.github/workflows/ruby.yml

```diff
@@ -0,0 +1,51 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ cdSetupBranch ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+      - name: Build with Gradle
+        uses: gradle/gradle-build-action@937999e9cc2425eddc7fd62d1053baf041147db7
+        with:
+          arguments: build
+      - name: Run server
+        run: |
+          ./gradlew run &
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5.1
+      - name: Install bundler
+        working-directory: ./http_server_spec
```

Comment:
Can you just `git pull` the `http_server_spec` repo rather than copying it into this code base?

Reply:
We discussed leaving this as is for now because I worked on it this morning without success